tests/module_selection_test.sh

```
@@ -0,0 +1,523 @@
+    fail "F-Droid cache was removed after locked verification"
+  assert_pair \
+    "--module-cache" "${FDROID_PRIVILEGED_EXTENSION_CACHE}" \
+    "F-Droid quoted cache"
+  assert_pair \
+    "--patch-report" "${FDROID_PRIVILEGED_EXTENSION_PATCH_REPORT}" \
+    "F-Droid quoted report"
+}
+
+test_fdroid_enabled_does_not_reuse_legacy_output_marker() {
+  reset_fixture fdroid-legacy-marker
+  ADDITIONALS[FDROID_PRIVILEGED_EXTENSION]="true"
```

Shellcheck (0.11.0)

[warning] ADDITIONALS appears unused. Verify use (or export if used externally).

(SC2034)